hw2/03-count.js

```diff
@@ -0,0 +1,11 @@
+// Add your code here
+let paragraphInfo = document.getElementById("paraInfo");
+let inputText = document.getElementById("userinput");
+inputText.addEventListener("input", (event) => {
+  const searchText = event.target.value;
+  const regex = new RegExp(searchText, "g");
+  let text = paragraphInfo.innerHTML;
+  text = text.replace(/(<mark>|<\/mark>)/gim, "");
+  const newText = text.replace(regex, "<mark>$&</mark>");
```

Comment:
You can add inline styling here to get the same yellow background as the screenshot in the requirements. It makes the JavaScript kind of ugly, and I know inline styling is in somewhat bad taste, but it's a lot more distinct than the default `<mark>` styling.

Just to be clear, this would take the form `<mark style="background-color: yellow;">`.

Reply:
Update the background color styling.